test/react-native-cli/features/fixtures/Dockerfile

```diff
@@ -2,14 +2,34 @@ FROM node:lts-alpine
 
 RUN apk add git
 
-COPY . /app
+# TODO(PLAT-5515) 'pod install' is currently always run but will only work on
+# macOS, so we stub it out with a script that does nothing
+RUN printf '#!/usr/bin/env sh\nprintf "this is a stub of $0\n"' > /usr/bin/pod && chmod +x /usr/bin/pod
 
 WORKDIR /app
 
-RUN npm i -g bugsnag-react-native-cli-*.tgz
+# Artifcatory authentication
+ARG REGISTRY_URL
+ARG REG_BASIC_CREDENTIAL
+ARG REG_NPM_EMAIL
+RUN npm config set registry $REGISTRY_URL
+RUN npm config set _auth $REG_BASIC_CREDENTIAL
+RUN npm config set email $REG_NPM_EMAIL
+RUN npm config set always-auth = true
 
-# TODO(PLAT-5515) 'pod install' is currently always run but will only work on
-# macOS, so we stub it out with a script that does nothing
-RUN printf '#!/usr/bin/env sh\nprintf "this is a stub of $0\n"' > /usr/bin/pod && chmod +x /usr/bin/pod
+# Files needs for installing the CLI
+COPY .git .git
```

Comment:
I guess we need this because of the `installCli` command? Can we remove it after or put it somewhere away from the fixtures (outside `/app`) as it will break the tests — the CLI checks if we're in a git repo and the tests assume that we're not unless it creates one

Reply:
Ah, ok. interesting and good spot.  I think there are a few approaches we could potentially take:
- Me find another way of passing the version in explicitly using `NOTIFIER_VERSION` (it's `determineVersion` that will use it)
- Remove the .git context in the docker file afterwards
- Create a Cucumber step to explicitly remove any git context it might find

I'm happy to look at the first, but I think the last also has some merit in terms of making the test robust.

Reply:
I think you could also copy this & run `installCli` somewhere other than `/app` for it to work — `installCli` runs globally so it doesn't have to happen in the same directory as the fixture projects

Reply:
Yeah, somehow I skipped over you telling me that in the initial comment.  I've moved it all into `/test` now, with just the fixture going into `/app`.